lib/pf/Authentication/Source.pm

```diff
@@ -97,7 +97,9 @@ sub common_attributes {
           { value => 'computer_name', type => $Conditions::SUBSTRING },
           { value => "mac", type => $Conditions::SUBSTRING },
           { value => "realm", type => $Conditions::SUBSTRING },
-         ];
+          { value => "switch", type => $Conditions::SUBSTRING },
```

Comment:
The switch in $args of pf::radius::switch_access is an object and cannot be used for string operations.
I would suggest that you use switch_id or switch_ip for this and fill it in as you did with switch_group.